src/Drive.php

```diff
@@ -287,17 +307,18 @@ public function getFile(string $path)
     }
 
     /**
-     * get file as a file resource
+     * Download the file at the given path in the current drive and return the stream
```

Comment:
```suggestion
     * Return a stream resource for the file at the given path in the current drive. The content of the file can be read from the stream.
```

Maybe text like this?